### DISCUSSION TITLE
Interactive Mouse Selection of Cuboid/2D Window with Foxglove: Is it Possible?

### DISCUSSION THREAD

**Discussion by arrfou99** [foxglove/community#639] [General]
Is it possible to allow the user to select a cuboid/2d window interactively using the mouse with Foxglove?  Like the rviz Interactive Markers where one could set a cuboid and change its dimension accordingly .http://wiki.ros.org/rviz/Tutorials/Interactive%20Markers%3A%20Getting%20Started

**Comment by jtbandes** [comment]
We don't currently have this feature, but we do have the ability to click and publish points and poses in the 3D panel (like rviz's "goal" setting functionality).
